plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -108,19 +103,21 @@ class Conan(
     private val config: ConanConfig
 ) : PackageManager("Conan") {
     companion object {
-        private val DUMMY_COMPILER_SETTINGS = arrayOf(
+        internal val DUMMY_COMPILER_SETTINGS = arrayOf(
             "-s", "compiler=gcc",
             "-s", "compiler.libcxx=libstdc++",
             "-s", "compiler.version=11.1"
         )
 
-        private const val SCOPE_NAME_DEPENDENCIES = "requires"
-        private const val SCOPE_NAME_DEV_DEPENDENCIES = "build_requires"
+        internal const val SCOPE_NAME_DEPENDENCIES = "requires"
+        internal const val SCOPE_NAME_DEV_DEPENDENCIES = "build_requires"
     }
 
+    internal val command by lazy { ConanCommand }
+
```

Comment:
As `ConanCommand` now still is an `object` in this commit, this code is unnecessary here.